src/language-js/postprocess.js

```diff
@@ -6,6 +6,10 @@ const { getLast } = require("../common/util");
 function postprocess(ast, options) {
   visitNode(ast, node => {
     switch (node.type) {
+      case "V8IntrinsicIdentifier":
+        node.name = `%${node.name}`;
+        node.type = "Identifier";
```

Comment:
Hmm, are we sure that we want to "lie" to all code that this is a regular identifier? Will this bite us in the future?

Reply:
I'm not 100% sure, but it's meant to work as an identifier for pretty much all purposes... Unless something relies on which chars it contains.

Reply:
So what's the conclusion? Can we proceed as-is? Note that even if there are some issues, they would appear only on code using such intrinsics, so in any case this is not regressing regular JavaScript.

Reply:
I chatted offline with @RReverser and he is going to send another version that doesn’t turn the intrinsic into an identifier during post processing but instead update the regular printer to handle intrinsics properly.

Once this is done, we should be good to accept.

On a side note, it’s really cool to see people at Google starting to play with prettier and contribute back. AMP and now v8 :)

Reply:
I've updated the PR to preserve node as-is. Unfortunately, there are still failures related to upgrade of `@babel/parser` - I'd be really grateful if that upgrade could be done by someone who is more familiar with the codebase :)